demisto_sdk/commands/format/update_generic_json.py

```diff
@@ -182,14 +180,14 @@ def run_format(self) -> int:
             self.save_json_to_destination_file()
             return SUCCESS_RETURN_CODE
         except Exception as err:
-            print(
+            logger.info(
```

Comment:
we could use `logger.exception` here

Reply:
https://github.com/demisto/demisto-sdk/pull/2917/commits/e10a99ec721e12b4ff94b9151b9f863d9ea62b83